public/app/features/explore/state/actions.ts

```diff
@@ -674,6 +685,27 @@ export function splitOpen(): ThunkResult<void> {
   };
 }
 
+/**
+ * Syncs time interval, if they are not synced on both panels in a split mode.
+ * Unsyncs time interval, if they are synced on both panels in a split mode.
+ */
+export function syncTimes(exploreId: ExploreId): ThunkResult<void> {
+  return (dispatch, getState) => {
+    if (exploreId === 'left') {
+      const leftState = getState().explore.left;
```

Comment:
I think we could instead just dispatch `updateTimeRange` action for the split we want to change.